### ISSUE TITLE
Tooltip on inputbox

### ISSUE THREAD

**Issue by ShahanaFarooqui** [Ride-The-Lightning/RTL-Design#40]
We want to add some detailed expalination on one of the inputs for loop-out like below screen. Is it ok to add tooltip directly on the input hover?

![image](https://user-images.githubusercontent.com/23240721/76128525-907b6c00-5fd2-11ea-92dc-4a6c6379a7e4.png)

**Comment by swedishfrenchpress** [comment]
Hey @ShahanaFarooqui, happy to chime in here. In general is it not good practice to introduce a tooltip directly on an input field. It's considered a better practice to hover over an icon indicating that there is more information that is NOT vital to task completion. Per NNgroup:
> Users shouldn’t need to find a tooltip in order to complete their task. Tooltips are best when they provide additional explanation for a form field unfamiliar to some users or reasoning for what may seem like an unusual request. Remember that tooltips disappear, so instructions or other directly actionable information, like field requirements, shouldn’t be in a tooltip. (If it is, people will have to commit it to their working memory in order to be able to act upon it.)

In the example above, if the info is not vital then we could add it to a tooltip. However, we should if possible make it brief. It's quite wordy now and risks adding information solution. I would purpose handling tooltips in a manner similar to this:
![image](https://user-images.githubusercontent.com/78821053/107986827-26828980-6fcd-11eb-8db1-5475b93a1bbf.png)

Open to feedback on the copy and tooltip design but I think we should aim to have some type of tooltip design standardised across the project. 

Resource: https://www.nngroup.com/articles/tooltip-guidelines/

**Comment by swedishfrenchpress** [comment]
@ShahanaFarooqui @saubyk 
I worked on some tool tip designs that will work on both light and dark mode. I modified the existing [material design info icon](https://material.io/resources/icons/?icon=restore_page&style=baseline) into a filled and outline version. The tool tip is a simple rectangle with a 3px rounded radius corners connected with a triangle. I'll included all the icons and tool tip bubbles in the export folder of my pull request. I hope this helps, open to any suggestions!

![Group 4](https://user-images.githubusercontent.com/78821053/108626022-5d81f080-744e-11eb-9db3-dfb23d998e99.png)

![Group 3](https://user-images.githubusercontent.com/78821053/108626021-5c50c380-744e-11eb-88d9-5e6b55b1199c.png)
